requirements/edx/github.txt

```diff
@@ -99,3 +99,4 @@ git+https://github.com/edx/xblock-lti-consumer.git@v1.0.3#egg=xblock-lti-consume
 # Third Party XBlocks
 -e git+https://github.com/mitodl/edx-sga@172a90fd2738f8142c10478356b2d9ed3e55334a#egg=edx-sga
 -e git+https://github.com/open-craft/xblock-poll@e7a6c95c300e95c51e42bfd1eba70489c05a6527#egg=xblock-poll
+-e git+https://github.com/open-craft/xblock-drag-and-drop-v2@3066fdbb5b002176226d15e2d1c3d33bc11939da#egg=xblock-drag-and-drop-v2
```

Comment:
Please follow the instructions at the top of the file.  If something about them is unclear, let me know so that we can clarify them.